packages/core/src/scp-cf/client-credentials-token-cache.ts

```diff
@@ -21,7 +22,13 @@ const ClientCredentialsTokenCache = (
     credentials: ClientCredentials,
     token: ClientCredentialsResponse
   ): void => {
-    cache.set(getGrantTokenCacheKey(url, credentials), token, token.expires_in);
+    cache.set(
+      getGrantTokenCacheKey(url, credentials),
+      token,
+      token.expires_in
```

Comment:
What does undefined mean? Is it valid forever or once? If once we don't need to cache it, I guess?

Reply:
since the cf doc doesn't tell whether this is mandatory, this might not be there.
as discussed, we don't use cache when it's undefined.

Reply:
Maybe we really just don't need to set it?